SparkleSpinTests/ModelTests.swift

```diff
@@ -18,30 +18,138 @@ class ModelTests: XCTestCase {
     override func tearDown() {
         // Put teardown code here. This method is called after the invocation of each test method in the class.
     }
+    
+    func testCreateOnePlayer() {
+        // given
+        let name = "Kevin"
+        
+        let playerListModel = PlayerListModel(playerList: [])
+        let playerListViewModel = PlayerListViewModel(playerListModel: playerListModel)
 
-    func testCreatePlayer() {
+        // when
+        let playerOne = playerListViewModel.createPlayerWith(name: name)
+
+        // then
+        XCTAssertEqual(playerOne.name, "Kevin")
+    }
+    
+    func testCreateTwoPlayers() {
+        // given
+        let nameOne = "Kevin"
+        let nameTwo = "Lisa"
+        
+        let playerListModel = PlayerListModel(playerList: [])
+        let playerListViewModel = PlayerListViewModel(playerListModel: playerListModel)
+        
+        // when
+        let playerOne = playerListViewModel.createPlayerWith(name: nameOne)
+        let playerTwo = playerListViewModel.createPlayerWith(name: nameTwo)
+        
+        // then
+        XCTAssertEqual([playerOne.name, playerTwo.name], ["Kevin", "Lisa"])
+    }
+    
+    func testAddOnePlayerToPlayerList() {
         // given
         let name = "Kevin"
-        let playerOneModel = PlayerModel(name: nil)
-        let playerOneViewModel = PlayerViewModel(playerModel: playerOneModel)
+        
+        let playerListModel = PlayerListModel(playerList: [])
+        let playerListViewModel = PlayerListViewModel(playerListModel: playerListModel)
+        let playerOne = playerListViewModel.createPlayerWith(name: name)
+        
+        // when
+        playerListViewModel.addPlayerToPlayerList(player: playerOne)
+        
+        // then
+        XCTAssertEqual(playerListModel.playerList?[0].name, "Kevin")
```

Comment:
here you could also do `(playerListModel.playerList?.first.name` I personally like to use `.first` because that helps to avoid crashes and is more idiomatic but you can always use either or just one more option :)

Reply:
Great idea, will do!